src/components/base/FormattedVal/index.js

```diff
@@ -56,6 +56,8 @@ type OwnProps = {
   disableRounding?: boolean,
   isPercent?: boolean,
   subMagnitude?: number,
+  prefix?: string,
+  suffix?: string,
```

Comment:
does that mean we can drop the one that was on Countervalue component and delegate to FormattedVal?

Reply:
No as this can only accept a string due to the fact it gets appended. The other one accepts React nodes :)